src/users/__tests__/userSelectors-test.js

```diff
@@ -124,3 +125,12 @@ describe('getUsersSansMe', () => {
     expect(getUsersSansMe(state)).toEqual([eg.otherUser]);
   });
 });
+
+describe('getUserIsActive', () => {
+  test('returns false for a user that has been deactivated', () => {
+    const users = [eg.makeUser(), eg.makeUser(), eg.makeUser()];
+    const deactivatedUser = eg.makeUser();
+    const state = eg.reduxState({ users });
+    expect(getUserIsActive(state, deactivatedUser.email)).toBeFalse();
+  });
```

Comment:
You also need to check that it returns `true` for a user who has *not* been deactivated! And, less obviously, for a bot (`realm.crossRealmBots`).

Although the current implementation doesn't rely on it, you should probably include `deactivatedUser` in the redux-state's `realm.nonActiveUsers`. Alternatively, you can just reuse the state setup from several of the tests above.

(Optional useful trick: you can set up a shared `state` in the `describe` block and use it in its sibling `test` blocks.)